searchform.php

```diff
@@ -1,8 +1,8 @@
 <form id="search" class="" method="get" action="<?php echo esc_url( home_url( '/' ) ); ?>"> 
   <div class="input-append">
 		<span aria-hidden="true" data-icon="&#xf002;"></span>
-		<input class="input-xlarge" type="text" name="s" value="<?php echo trim( get_search_query() ); ?>">
+		<input role="search" label="Search" class="input-xlarge" type="text" name="s" value="<?php echo trim( get_search_query() ); ?>"/>
```

Comment:
See notes on other pull request around incorrect usage of role

Reply:
resolved in commit.